tests/clients/layer0_test_client.go

```diff
@@ -22,81 +25,114 @@ func NewLayer0TestClient(t *testing.T, endpoint, token string) *Layer0TestClient
 	}
 }
 
-func (l *Layer0TestClient) CreateTask(taskName, environmentID, deployID string, copies int, overrides []models.ContainerOverride) string {
-	jobID, err := l.Client.CreateTask(taskName, environmentID, deployID, copies, overrides)
+func (l *Layer0TestClient) CreateTask(req models.CreateTaskRequest) string {
+	jobID, err := l.Client.CreateTask(req)
```

Comment:
[nit] this could be more DRY with a helper:
```go
func (l *Layer0TestClient) CreateTask(req models.CreateTaskRequest) string {
	return l.jobHelper(func() (string, error) {
		return l.Client.CreateTask(req)
	})
}

func (l *Layer0TestClient) jobHelper(fn func() (string, error)) string {
	jobID, err := fn()
	if err != nil {
		t.Fatal(err)
	}

	return job.Result
}
```